articles/active-directory/develop/quickstart-v2-javascript.md

```diff
@@ -37,7 +37,9 @@ In this quickstart, you use a code sample to learn how a JavaScript single-page
 > 1. Sign in to the [Azure portal](https://portal.azure.com) by using either a work or school account, or a personal Microsoft account.
 > 1. If your account gives you access to more than one tenant, select the account at the top right, and then set your portal session to the Azure Active Directory (Azure AD) tenant you want to use.
 > 1. Go to the new [Azure portal - App registrations](https://portal.azure.com/#blade/Microsoft_AAD_RegisteredApps/ApplicationsListBlade/quickStartType/JavascriptSpaQuickstartPage/sourceType/docs) pane.
-> 1. Enter a name for your application, and select **Register**.
+> 1. Enter a name for your application. 
+> 1. Under **Supported account types**, select **Accounts in any organizational directory and personal Microsoft accounts**.
+> 1. Select **Register**.
 > 1. Follow the instructions to download and automatically configure your new application.
```

Comment:
I think you should direct them to go to the quickstart in the portal for this step

Reply:
hmm.. there seems to be no instruction on the portal until you complete the app registration following that link